compiler/aarch64/codegen/OpBinary.cpp

```diff
@@ -309,8 +311,8 @@ const OMR::ARM64::InstOpCode::OpCodeBinaryEntry OMR::ARM64::InstOpCode::binaryEn
 		0xD3400000,	/* UBFM      	ubfmx	 */
 	/* Extract */
 		0x13800000,	/* EXTR      	extrw	 */
-		0x93C00000,	/* EXTR      	extrx	 */
-/* Data Processing - register */
+		0x93C08000,	/* EXTR      	extrx	 */
+/* Data Processing – register */
```

Comment:
Looks like you missed this hyphen, n-dash, m-dash etc. The other one looks to be fixed now.